src/ViewListingDisplay/components/Listing.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import PostCommentsDisplay from '../../PostCommentsDisplay/components/PostCommentsDisplay.jsx';
+
+import '../styles/Listing.css';
+
+/** Functional Component that creates a listing. Takes in listingInfo as a prop 
```

Comment:
same with 'c'. Also props.

Multi line comment should be 
/** Here is my comment. Passing to
 * next line. 
 */

Reply:
Done